projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,128 @@
+
+ 
+Mendix has been a key component in making the Preparese<sup>TM</sup> solution possible. Low-code software development has been growing in popularity and bringing new audiences to the practice. Mendix, one of the more popular platforms with over 300K active developers and 50M users, has enabled Entidad and the UFW Foundation to develop, launch, and maintain enterprise-grade digital trust enabled solutions that solve real world problems. 
+
+ 
+We’ve built two products, using this technology. For organizations we’ve built Preparese Platform<sup>TM</sup>, it allows them to quickly develop and launch digital trust enabled services and programs. For farmworkers, we’ve developed Preparese Mobile<sup>TM</sup> which combines a verifiable digital profile with a suite of tools that simplify interactions with organizations on the platform. 
+
+ 
+The Preparese Platform<sup>TM</sup> is being used by 9 organizations and has 5 digital services in production. The most recent service launched is being used to process and distribute [$80 million](https://www.usda.gov/media/press-releases/2021/09/07/usda-invests-700-million-grants-provide-relief-farm-and-food) in one-time relief payments to over 125,000 workers, under the United States Department of Agriculture’s [Farm and Food Workers Relief Program](https://www.ams.usda.gov/services/grants/ffwr) (FFWR).  
+
+  
+The second product, Preparese Mobile<sup>TM</sup>, is designed around the unique needs of farmworkers. We’ve developed a react native mobile app that lets farmworkers store, manage, and exchange their verified information. Engagement with organizations is made easier with [DIDComm](https://didcomm.org/book/v2/)-based communication capabilities such as text and video chat.  
+
+ 
+One of the cornerstone technologies of the solution is an interoperable Mendix enabled digital wallet. The wallet components need to be able to engage with various non-profit, government, and philanthropic sources, each with their own technology stacks. For this reason, interoperability and working with open standard frameworks has been a priority. The project team members have participated in [TrustOverIP Foundation](https://trustoverip.org/), [Hyperledger Aries JS Working Group](https://wiki.hyperledger.org/display/ARIES/Aries+Framework+JavaScript), and various other communities, including [DID Communication Working Group](https://identity.foundation/working-groups/did-comm.html) and [OpenWallet Foundation](https://openwallet.foundation/) of late, to ensure we adhere to best practices and standards.       
```

Comment:
Which interoperable protocols and credential formats does this wallet support today? What additional protocols and credential formats are planned?

Reply:
`AnonCreds` credential format and `DIDComm v1` are currently supported. The project roadmap includes adding support for DIDComm v2 and W3C VC including:
- `OpenID4VC`
- `W3C JSON-LD`
- `SD-JWT`